Horion/Module/Modules/TriggerBot.cpp

```diff
@@ -16,20 +17,37 @@ const char* TriggerBot::getModuleName()
 {
 	return ("TriggerBot");
 }
-
+static std::vector <C_Entity*> targetList0;
 void TriggerBot::onTick(C_GameMode* gm)
 {
 	C_LocalPlayer* localPlayer = g_Data.getLocalPlayer();
 	C_Entity* target = g_Data.getClientInstance()->getPointerStruct()->entityPtr;
 	Odelay++;
 	if (target != 0 && Odelay >= delay)
 	{
+		if (target == g_Data.getLocalPlayer()) // Skip Local player
+			return;
+
+		if (target == 0)
+			return;
+
+		if (target->timeSinceDeath > 0 || target->damageTime >= 7)
```

Comment:
What has this to do with friends

Reply:
friends and valid entities.... trigger bot used to just smack anything

Reply:
it used to hit npc and shit in servers now it only hits players that are not your friend

Reply:
Add AntiBot module and build in check to Aura and Triggerbot? Would have more use

Reply:
someone else can do that. I just edited trigger bot on request from a guy in discord.